README.md

```diff
@@ -26,26 +26,13 @@ Adds a set of popular Google Web Fonts to Design Palette Pro. Provides the prope
 
 ## Frequently Asked Questions ##
 
-### What is this? ###
-
-The description covers it all.
-
 ### Why Doesn't This Work On My Theme? ###
 
 Because this is an add-on for [Design Palette Pro](http://genesisdesignpro.com/ "Genesis Design Palette Pro"). Without that plugin, nothing will work.
 
 ### How Many Fonts Are Included? ###
 
-There are 64 total fonts included
-
-### Why Aren't All Fonts Included? ###
-
-Because having a list of 800+ fonts would be very difficult (if not impossible) to actually use.
-
-### What Fonts Are Included? ###
-
-* [All Google fonts](https://fonts.google.com/) are now available!
-
+As of version 2.0, [all 900+ Google Fonts](https://fonts.google.com/) are included.

```

Comment:
This might need a qualification like 
```suggestion
As of version 2.0, [all 900+ Google Fonts](https://fonts.google.com/) are included when used in conjunction with Design Palette Pro 1.8+.
```

If someone updates and has 1.7- they will still have access to the original font stack.

Reply:
sorry, already in 1.8 dev mode in my head :D 

Yes, 1.7+

Reply:
Updated my comment if you want to just commit the suggestion.